js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -808,3 +808,21 @@ invalidateRotatedSecret=Das rotierte Secret invalidieren?
 noDirectUsers=Keine direkten Benutzer
 permissionsDisable=Berechtigung abschalten?
 eventTypes.PERMISSION_TOKEN.name=Berechtigungstoken
+eventTypes.UPDATE_TOTP.name=Update totp
```

Comment:
@robson90 - could you please create a PR for the English text that it is `Update TOTP`, as TOTP is an acronym for a time-based one-time password. In parallel, I suggest to name this `Aktualisierung TOTP`.